allensdk/brain_observatory/ecephys/write_nwb/__main__.py

```diff
@@ -697,6 +703,20 @@ def write_ecephys_nwb(
     add_running_speed_to_nwbfile(nwbfile, running_speed)
     add_raw_running_data_to_nwbfile(nwbfile, raw_running_data)
 
+    if eye_dlc_ellipses_path and eye_gaze_mapping_path:
```

Comment:
is there a case where we want to write ellipses but not gaze mapping?

Reply:
No, because at least currently the eye_dlc_ellipses h5 doesn't have any synced timestamps. Only the gaze mapping data has that information. And without the synced timestamps, I would argue the eye ellipses are worthless as well...